scripts/addFromConfig.py

```diff
@@ -256,7 +250,7 @@ def addStuff(cfg, options):
             # now add the productprocesslink
             tmp = dict((k, cfg[p][k]) for k in cfg[p] if 'input' in k)
             for k in tmp:
-                ppl = dbu.addproductprocesslink(cfg[tmp[k][0]]['product_id'], p_id, 'optional' in k, tmp[k][1], tmp[k][2])
+                ppl = dbu.addproductprocesslink(cfg[tmp[k][0]]['product_id'], p_id, 'optional' in k) 
```

Comment:
I think this completely destroys support for yesterday/tomorrow; what's happening here?

Reply:
Not sure yet.  It's been a while, my guess is that I was getting wrong answers and it just did not work.

Reply:
So I think we remove this then and assume the problem was elsewhere?